src/components/MultiStepIndicator/components/StepIndicator/StepIndicator.tsx

```diff
@@ -139,18 +170,18 @@ const StepIndicator = ({
         <div
           className={cx(...getClassNamesWithSuffix("__number-container"))}
           ref={componentRef}
-          tabIndex="0"
+          tabIndex={0}
           role="button"
         >
           <SwitchTransition mode="out-in">
-            <CSSTransition
+            <CSSTransition<undefined> /* https://stackoverflow.com/questions/64019519/typing-out-react-transition-group-handlers-in-typescript */
```

Comment:
mmmmm Can you explain?

Reply:
CSSTransition needs to be specified with the generic parameter to decide type for callback (out of few possible)
We can add instead ts-ignore, but if we don't specifiy the generic param the callback `addEndListener` expects only one argument (of `done`, without `node`)

Reply:
<img width="749" alt="image" src="https://github.com/mondaycom/monday-ui-react-core/assets/30905362/b801ba6c-94d0-44c5-85fc-355c44bc4251">

Reply:
Move the comment to be above the <CssTran..> and remove stackoverflow link - explain in plain text